htdocs/comm/propal/card.php

```diff
@@ -1766,6 +1766,7 @@
 	$fk_account = GETPOST('fk_account', 'int');
 
 	// Load objectsrc
+	$objectsrc = NULL;
```

Comment:
Try to use the syntax
`$objectsrc = null;`

Reply:
Thanks, corrected it